_includes/head.html

```diff
@@ -1,11 +1,22 @@
 <head>
+
+    <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <link rel="icon" type="image/png" href="./img/logo/logoOA_svg.png">
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width initial-scale=1" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-
-    <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <meta name="keywords" content="foss, open source, organisations, 2018, technologists, astronomy,
```

Comment:
```suggestion
     <meta name="keywords" content="foss, open source, organisations, astronomy,
```

Reply:
I deleted 2018 and technologists from keywords.